### PR TITLE
Moccal tests

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Adds gtests for moccal 

## Motivation and Context

Part of removing old tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by krlberry — CHANGES_REQUESTED** [review]
Could you add tests for the calibration group that is added to the label. Otherwise, looks great!

File: isis/tests/data/mgsImages/Testing/Temporary/CTestCostData.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (krlberry) [inline isis/tests/data/mgsImages/Testing/Temporary/CTestCostData.txt]: Did you mean to add this file?

File: isis/tests/data/mgsImages/Testing/Temporary/LastTest.log [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Start testing: Feb 26 11:54 MST
```
Comment (krlberry) [inline isis/tests/data/mgsImages/Testing/Temporary/LastTest.log]: Did you mean to add this file?

File: isis/tests/data/mgsImages/Testing/Temporary/CTestCostData.txt (hunk 1 above)
Comment (Kelvinrr) [reply isis/tests/data/mgsImages/Testing/Temporary/CTestCostData.txt]: that is so bizarre, I'll remove them

**Review by krlberry — APPROVED** [review]

**Pull request merged**